pkg/apis/litmuschaos/v1alpha1/chaosengine_types.go

```diff
@@ -103,9 +103,9 @@ type ExperimentAttributes struct {
 
 // ExperimentComponents contains ENV, Configmaps and Secrets
 type ExperimentComponents struct {
-	ENV        []ExperimentENV        `json:"env,omitempty"`
-	ConfigMaps []ExperimentConfigMaps `json:"configMaps,omitempty"`
-	Secrets    []ExperimentSecrets    `json:"secrets,omitempty"`
+	ENV        []ExperimentENV `json:"env,omitempty"`
+	ConfigMaps []ConfigMap     `json:"configMaps,omitempty"`
```

Comment:
@ksatchit @chandankumar4 Why are we using capitals, in json spec? Shouldn't we go with `configmaps`, this will keeps us in sync with chaosExperiment configmaps